src/libraries/System.Diagnostics.PerformanceCounter/src/System/Diagnostics/PerformanceCounterLib.cs

```diff
@@ -1112,12 +1112,12 @@ private Hashtable GetStringTable(bool isHelp)
                             if (isHelp)
                             {
                                 // Category Help Table
-                                throw new InvalidOperationException(SR.Format(SR.CategoryHelpCorrupt, names[index * 2]));
+                                throw new InvalidOperationException(SR.Format(SR.CategoryHelpCorrupt, names[index * 2], _machineName));
                             }
```

Comment:
is there any concern mentioning the machine names in the messages which can be exposed in the logs?

Reply:
Good point - we include info like this in many messages already, and @shirhatti is investigating whether we should offer a facility to scrub all messages. Meantime, I'll take the machine name out.

Reply:
updated